### DISCUSSION TITLE
Validator `once` method - extending the validator for validating single values

### DISCUSSION THREAD

**Discussion by RichToms** [laravel/framework#38960] [Ideas]
The validator has a lot of great rules built into it that can make typical development and checks much easier to reading/writing long if statements for validation

I propose a `once` method to the Validator that allows you to pass a single value and an array/string of rules, the result is a boolean of whether the value passes those rules or not. This could also accept custom Rules as the usual Validator can.

## Before
```php
if (is_int($someValue) && $someValue >= 0 && $someValue < 10) {
```

## After
```php
if (Validator::once($someValue, ['numeric', 'between:0,10'])) {
```

**Comment by dennisprudlo** [comment]
You can use the global helper.

```php
validator([
    'field' => $someValue
], [
    'field' => ['numeric', 'between:0,10']
])->passes();
```

Sure, you would still have to name the field, but there's no `is_int`, `>=` and `<` involved.

**Reply by RichToms** [reply]
That is true, but the syntax of that is what I am proposing to avoid having to do - being able to utilise those rules in your selections will make them easier to read complex selections as the rules are quite clear
